packages/form-plugin/tests/form.plugin.spec.ts

```diff
@@ -645,5 +667,123 @@ describe('NgxsFormPlugin', () => {
       // Assert
       expect(statuses).toEqual(['INVALID', 'VALID']);
     });
+
+    it('should reset form when ResetForm action call', () => {
+      // Arrange
+      @State({
+        name: 'todos',
+        defaults: {
+          todosForm: {
+            model: {
+              text: 'Buy some coffee'
+            },
+            dirty: true
+          }
+        }
+      })
+      @Injectable()
+      class TodosState {}
+
+      @Component({
+        template: `
+          <form [formGroup]="form" ngxsForm="todos.todosForm">
+            <input formControlName="text" /> <button type="submit">Add todo</button>
+          </form>
+        `
+      })
+      class MockComponent {
+        public form = new FormGroup({
+          text: new FormControl()
+        });
+      }
+
+      // Act
+      TestBed.configureTestingModule({
+        imports: [
+          ReactiveFormsModule,
+          NgxsModule.forRoot([TodosState]),
+          NgxsFormPluginModule.forRoot()
+        ],
+        declarations: [MockComponent]
+      });
+
+      const store = getStore();
+      const fixture = TestBed.createComponent(MockComponent);
+
+      fixture.detectChanges();
+
+      store.dispatch(new ResetForm({ path: 'todos.todosForm' }));
+
+      fixture.detectChanges();
+
```

Comment:
Also please place the `// Arrange` on this line, because the `input` variable is only required for the assertions and is not directly part of the action we are testing.